src/clarity-angular/modal/modal.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2016 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {
+    Component,
+    Input,
+    Output,
+    EventEmitter,
+    HostListener,
+    OnChanges,
+    OnDestroy,
+    SimpleChange,
+    animate,
+    style,
+    transition,
+    trigger
+} from "@angular/core";
+import {ScrollingService} from "../main/scrolling-service";
+
+@Component({
+    selector: "clr-modal",
+    viewProviders: [ScrollingService],
+    templateUrl: "./modal.html",
+    styles: [`
+        :host { display: inline-block; }
+    `],
+    animations: [
+        trigger("fadeDown", [
```

Comment:
Regarding this, I'm fine pushing with duplicate hard-coded animations for now, as long as we create an issue to fix it before we forget.